src/editor.cls.php

```diff
@@ -0,0 +1,155 @@
+<?php
+/**
+ * The Editor class
+ *
+ * @since
+ * @package     LiteSpeed
+ * @subpackage  LiteSpeed/inc
+ * @author      LiteSpeed Technologies <info@litespeedtech.com>
+ */
+namespace LiteSpeed;
+
+defined( 'WPINC' ) || exit;
+
+class Editor extends Base {
+
+  /**
+  * Add a post meta box into the sidebar of post editor.
+  *
+  * @since    4.3
+  */
+  function add_meta_boxes() {
+    Debug2::debug( '[Editor] add_meta_boxes ');
+    add_meta_box(
+      'litespeed_metabox',                      // Unique ID
+      'Litespeed Cache',                        // Title
+      array( $this, 'meta_box_html' ),          // Callback function
+      null,                                     // Admin page (or post type)
+      'side',                                   // Context
+      'high'                                    // Priority
+    );
+  }
+
+  /**
+  * Display the post meta box.
+  *
+  * @since    4.3
+  */
+  function meta_box_html( $post ) {
+    if ( false ) {
+      $post_meta = get_post_meta( $post->ID, 'litespeed_metabox_vpi', true );
```

Comment:
As the following lines are only used in tpl file, why don't put all of them into tpl file then :) ?